aiidalab_launch/__main__.py

```diff
@@ -45,6 +45,11 @@
 
   ssh {user}@{hostname} -NfL {port}:localhost:{port}
 
+If you visit the remote machine through a jump server, you run (replace the J_IP with the jump server IP):
+
+  ssh -J {user}@J_IP {user}@{hostname} -NfL {port}:localhost:{port}
```

Comment:
Is that possible to get the `J_IP` if you are already ssh to the remote through SSH, this command is suppose to be copy-paste and run.